### DISCUSSION TITLE
can you make a video tutorial?

### DISCUSSION THREAD

**Discussion by elliotswinning** [Hyperclaw79/PokeBall-SelfBot#218] [Q&A] [answered]
can you please make a video tutorial? i dont understand it

**Comment by Hyperclaw79** [comment, marked as answer]
It already exists for the Premium Version.

**Reply by elliotswinning** [reply]
how do i get the premium version?

**Reply by Hyperclaw79** [reply]
You can get it by DMing me on Discord. It's priced at $25 USD and for more details, check out the Donations section of the Readme.

**Reply by elliotswinning** [reply]
how would i pay? paypal

**Comment by Hyperclaw79** [comment]
Yes, it's through PayPal but first DM me on discord so that I can create an
invoice for you to pay from.

**Comment by OraxCG** [comment]
Whats ur discord. i would like to write with you etc

**Reply by Hyperclaw79** [reply]
i'm currently not using discord for communications. Premium version users can converse with me via mail.

**Reply by OraxCG** [reply]
okay, since i tried to setup the bot but didnt quite work when i tried to follow ur step. soo was hoping if u could try to explain me what was wrong. But how much is for premium?

**Comment by Hyperclaw79** [comment]
Please read the Closed Issues and the Readme.
Public version code will not work as it is 3yrs old and unmaintained.
Currently, only Premium version is maintained and works without any issues.

**Reply by OraxCG** [reply]
ah okay
